migrator.go

```diff
@@ -0,0 +1,33 @@
+package pspmigrator
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"k8s.io/pod-security-admission/api"
+	psaApi "k8s.io/pod-security-admission/api"
+	"k8s.io/pod-security-admission/policy"
+)
+
+func SuggestedPodSecurityStandard(pod *v1.Pod) (psaApi.Level, error) {
+	evaluator, err := policy.NewEvaluator(policy.DefaultChecks())
+	if err != nil {
+		return "", err
+	}
+	apiVersion, err := api.ParseVersion("latest")
+	if err != nil {
+		return "", err
+	}
+	for _, level := range []string{"restricted", "baseline"} {
+		apiLevel, err := psaApi.ParseLevel(level)
+		if err != nil {
+			return "", err
+		}
```

Comment:
nit: just iterate over the level constants and skip the parsing:
```suggestion
	for _, apiLevel := range []string{psaApi.LevelRestricted, psaApi.LevelBaseline} {
```

Reply:
much cleaner. Thanks!